frontend/src/metabase/public/components/widgets/SharingPane/SharingPaneButton/SharingPaneButton.styled.tsx

```diff
@@ -0,0 +1,82 @@
+import styled from "@emotion/styled";
+import type { HTMLAttributes } from "react";
+import { css } from "@emotion/react";
+import { PublicEmbedIconWrapper } from "metabase/public/components/widgets/SharingPane/icons/PublicEmbedIcon/PublicEmbedIcon.styled";
+import { StaticEmbedIconWrapper } from "metabase/public/components/widgets/SharingPane/icons/StaticEmbedIcon/StaticEmbedIcon.styled";
+import type { PaperProps, ButtonProps } from "metabase/ui";
+import { Button, Paper, Title } from "metabase/ui";
+
+export const SharingPaneButtonTitle = styled(Title)``;
+export const SharingPaneActionButton = styled(Button)<
+  ButtonProps & HTMLAttributes<HTMLButtonElement>
+>``;
+
+export const SharingPaneButtonContent = styled(Paper)<
+  PaperProps & { disabled?: boolean }
+>`
+  ${({ theme }) =>
+    css`
+      ${StaticEmbedIconWrapper} {
+        rect.outerFill {
+          stroke: ${theme.colors.bg[1]};
+        }
+
+        path.outerFill {
+          fill: ${theme.colors.bg[1]};
+        }
+
+        .innerFill {
+          fill: ${theme.colors.text[2]};
+        }
+      }
+
+      ${PublicEmbedIconWrapper} {
+        .outerFill {
+          stroke: ${theme.colors.bg[1]};
+        }
+
+        .innerFill {
+          stroke: ${theme.colors.text[2]};
+        }
+      }
+    `}
+
+  ${({ disabled, theme }) =>
+    !disabled &&
+    css`
+      &:hover {
```

Comment:
Maybe you could check with our product or designer, but I think the button should only be blue when hovering on the button itself, not on the card. I find it feels a little bit weird when the button turned blue even when I'm not hovering on it.

Reply:
@oisincoveney I just looked at [the prototype](https://www.figma.com/proto/6cHI5jaztfjDYvEXEhy9uE/Setup-flow-for-static%2Fpublic-embedding-%26-embedding-feature-discoverability?type=design&node-id=129-2053&t=3Tb9SQejXymI8wCi-0&scaling=min-zoom&page-id=129%3A1872&starting-point-node-id=129%3A2050&show-proto-sidebar=1), when hovering over the whole card the cursor would change to a click icon, that hints the design meant the entire card would be clickable, not just the button. If that's the case, then it makes sense for the button background to change when hovering over the card.